CMakeLists.txt

```diff
@@ -230,6 +261,7 @@ find_package(ZLIB)
 if(ZLIB_FOUND)
 	set(HAVE_LIBZLIB 1)
 	list(APPEND ZM_BIN_LIBS "${ZLIB_LIBRARIES}")
+	set(LIBS "${LIBS} -lz")
```

Comment:
Need to confirm this 100% but @connortechnology was asking about these. I believe they are to let plugins know what libs are present. libzoneminder is related just need to put it all together.